crates/next-core/js/src/dev/hmr-client.ts

```diff
@@ -116,6 +116,15 @@ function aggregateUpdates(
   const key = resourceKey(msg.resource);
   const aggregated = chunksWithUpdates.get(key);
 
+  if (msg.type === "issues" && aggregated == null && hasIssues) {
+    // add an empty record to make sure we don't call `onBuildOk`
+    chunksWithUpdates.set(key, {
```

Comment:
We check if there are entries in the map below, so even if we have no updates we insert an empty record to make sure it doesn't think everything is fine